.github/workflows/publish-latest.yml

```diff
@@ -1,10 +1,11 @@
 # Description: this workflow publishes UI packages to `latest` tag on NPM.
 #
-# Triggered by: whenever "Version Packages" is merged
+# Triggered by: whenever "Version Packages" is merged to `main`
 
 # If successful, we will publish all updated UI packages to
 #   - https://www.npmjs.com/package/@aws-amplify/ui
 #   - https://www.npmjs.com/package/@aws-amplify/ui-react
+#   - https://www.npmjs.com/package/@aws-amplify/ui-react-native
```

Comment:
Smallest nit in the world: but if we're updating this list of packages this is technically missing the categorical packages right?

Reply:
Think this comment is pretty outdated now, maybe we just say

> If successful, we will publish all updated UI packages to NPM.